README.md

```diff
@@ -0,0 +1,197 @@
+If these are all good, you can check:
+
+- Rails (rails --version)
+
+If you get a good version, you're ready to go! (Almost).
+If you didn't, you may still need to install the Rails gem.
+
+`gem install rails`
+
+It can be good to keep in mind which version you're using as each of these will update on their own schedule and sometimes the code changes just enough to give us bugs. This will be true for all the packages, gems, APIs, or any other dependencies you may bring into your projects as a developer.
+
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone https://github.com/ZuhRa-HashiMi/budget_app
+```
+
+### Run
+
+To run this app on your local environment you need to run `bundle install`
+
+### Usage
+
+You need to get your Rails application server running,in order to see your app in the browser. This is like live preview in the browser for rails applications. Run :
+
+
+```sh
+ $ bin/rails server
+
+```
+
+OR
+
+```sh
+$ rails s
+
+```
+
+visit [localhost](http://localhost:3000/) in your browser!
+
+To exit server enter “Control (^ )+ C”
+
+### Tests
+
+To create tests, RSpec testing framework is used. In order to run tests use following command:
+
+
+```sh
+ $ rspec spec
+
+```
+_Note: You can also run the test by using just `rspec` but it runs all tests folders and files with name like `****_spec`_
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- Live Demo -->
+
+## **Live Demo**
+
+- [🚀 Live Demo](https://budget-app-k2i1.onrender.com)
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Zuhra Hashimi**
+
+- GitHub: [@ZuhRa-HashiMi](https://github.com/ZuhRa-HashiMi)
+- LinkedIn: [Zuhra Hashmi](https://www.linkedin.com/in/zuhra-hashimi/)
+- AngelList: [zuhra-hashimi](https://wellfound.com/u/zuhra-hashimi)
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Improving UI**
+- [ ] **Make an API of this app**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project please give this repo a star ⭐️
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+Project architecture and requirments provided by Microverse
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
Please provide a valid MIT license link; the current link is broken 😉 
![Screenshot from 2023-04-14 17-51-09](https://user-images.githubusercontent.com/44624138/232110186-64d9acb2-0846-4a1f-b561-85b4e9c02162.png)